pages/validators/_address.vue

```diff
@@ -38,6 +38,12 @@
             type="secondary"
             @click.native="openUnstakeModal"
           />
+          <CommonButton
+            :disabled="!delegation"
+            :value="`Restake`"
```

Comment:
```suggestion
            value="Restake"
```

Reply:
The original author uses the same coding style like:
```
<CommonButton :value="`Stake`" @click.native="openStakeModal" />
<CommonButton
  :disabled="!delegation"
  :value="`Unstake`"
  type="secondary"
  @click.native="openUnstakeModal"
/>
```
Is it preferred to follow?

Reply:
Then follow their style